ledger/error/src/main/scala/com/daml/error/ErrorCategory.scala

```diff
@@ -332,8 +333,26 @@ object ErrorCategory {
       )
       with ErrorCategory
 
-  implicit val orderingErrorType: Ordering[ErrorCategory] = Ordering.by[ErrorCategory, Int](_.rank)
+  @Description(
+    """This error category is used to signal that an unimplemented code-path has been triggered by a client or participant operator request."""
+  )
+  @RetryStrategy("""Errors in this category are non-retryable.""")
+  @Resolution(
+    """This error is caused by a ledger-level misconfiguration or by an implementation bug.
+      |Resolution requires ledger operator intervention or vendor support."""
```

Comment:
```suggestion
      |Resolution requires participant operator intervention."""
```

The operator can then escalate to the vendor if required.